src/hooks/index.ts

```diff
@@ -15,6 +15,7 @@ export {
   useList,
   useListRow,
   useGrid,
+  resetToTop,
```

Comment:
Hmm..no, I think should exist in both `useGrid` and `useList` but not a standalone `resetToTop` as you can access both ref in those hooks directly. But your standalone requires a pass..
While the logic does work it's a bit out of character for the flow

Reply:
Updated.